src/elements/common/empty-state/EmptyState.scss

```diff
@@ -4,4 +4,5 @@
     flex: 1;
     flex-direction: column;
     justify-content: center;
+    text-align: center;
```

Comment:
Unrelated?

Reply:
unrelated but fixes text alignment for the empty state at small widths. I can make this a separate change if you'd like.